packages/calcite-components/src/components/card/card.scss

```diff
@@ -3,18 +3,24 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ *
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
  * @prop --calcite-card-background-color: Specifies the background color of the component.
  * @prop --calcite-card-border-color: Specifies the border color of the component.
- * @prop --calcite-card-shadow: Specifies the shadow of the component.
+ * @prop --calcite-card-checkbox-background-color-checked: defines the background-color of the sub-component when checked.
```

Comment:
I’d honestly be ok with not adding these checkbox values at all. The property associated with them on the Card was deprecated with the introduction of the Card Group. It wasn’t possible to theme this explicitly before so we shouldn’t have to worry about maintaining any existing behavior. The Card component has new css props for the equivalent theming need.